src/tracks/Pattern.cpp

```diff
@@ -1196,15 +1198,11 @@ void PatternView::paintEvent( QPaintEvent * )
 
 	p.setFont( pointSize<8>( p.font() ) );
 
-	QColor text_color = ( m_pat->isMuted() || m_pat->getTrack()->isMuted() )
-		? QColor( 30, 30, 30 )
-		: textColor();
-
 	if( m_pat->name() != m_pat->instrumentTrack()->name() )
 	{
-		p.setPen( QColor( 0, 0, 0 ) );
+		p.setPen( textShadowColor() );
 		p.drawText( 4, p.fontMetrics().height()+1, m_pat->name() );
-		p.setPen( text_color );
```

Comment:
Before and After: 
![screenshot from 2016-02-19 19 36 42](https://cloud.githubusercontent.com/assets/6282045/13185221/3fd627aa-d740-11e5-9b87-d6edc6a7431d.png) ![screenshot from 2016-02-19 19 37 21](https://cloud.githubusercontent.com/assets/6282045/13185220/3fd57bc0-d740-11e5-8dad-89529cc37892.png)